src/components/DoctorsPage/DoctorsCard/Schedule/ScheduleItem/ScheduleItem.tsx

```diff
@@ -26,8 +29,9 @@ const ScheduleItem: React.FC<IScheduleItemProps> = ({
     dispatch({
       type: ScheduleActionTypes.SET_CHOICE_WORK_TIME,
```

Comment:
you should call a separate action here instead of dispatching actions directly from the component

Reply:
I don't quite understand what should be

Reply:
fixed, i moved to actionCreator